modules/op-results-reference-results.adoc

```diff
@@ -0,0 +1,45 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-results-for-openshift-pipelines-observability.adoc
+
+:_content-type: REFERENCE
+[id="results-reference-results_{context}"]
+= Reference information for searching results
+
+You can use the following fields in CEL queries for results:
+
+.Fields available in CEL queries for results
+|===
+| CEL field | Description
+
+| `parent`
+| The namespace in which the `PipelineRun` custom resource (CR) was created.
```

Comment:
```suggestion
| The namespace in which the `PipelineRun`/`TaskRun` custom resource (CR) was created.
```

Reply:
@avinal how can this be a TaskRun? We are talking of a Result, not a Record. I thought a Result was only created for a pipeline run?